snuba/subscriptions/codecs.py

```diff
@@ -32,7 +33,7 @@ def decode(self, value: bytes) -> SubscriptionData:
         except json.JSONDecodeError:
             raise InvalidQueryException("Invalid JSON")
 
-        return SubscriptionData.from_dict(data, self.entity_key)
+        return SnQLSubscriptionData.from_dict(data, self.entity_key)
```

Comment:
Shouldn't this be just `SubscriptionData`?

Reply:
`SubscriptionData` is the base abstract class now